src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -104,10 +104,11 @@
               (fn [acc e]
                 (render-inline acc e chat-id style-override mention-first))
               [quo/text
-               {:style {:size          :paragraph-1
-                        :margin-bottom (if mention-first (if platform/ios? 4 0) 2)
+               {:size  :paragraph-1
+                :style {:margin-bottom (if mention-first (if platform/ios? 4 0) 2)
                         :margin-top    (if mention-first (if platform/ios? -4 0) -1)
-                        :color         (when (seq style-override) colors/white)}}]
+                        :color         (when (seq style-override) colors/white)
+                        :line-height   22.75}}]
```

Comment:
could you please elaborate why we have custom line height here ?

Reply:
From figma design, I calculated the line height.

Reply:
If I'm looking at the correct [figma design](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6166-62737&mode=design&t=GbZ3g1O5BJCSq9ov-4), the `line-height` is `21.75` which is the exact `line-height` of the [`:paragraph-1`](https://github.com/status-im/status-mobile/blob/0a8993bbf19fc368e4f3f04ed5daf73062712203/src/quo2/foundations/typography.cljs#L29) text.

I also noticed that Figma uses [Message Type](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=3188-49964&mode=design&t=BwbRgofwzOmOkKrp-4) component from the design system that we haven't implemented in code yet.